src/Microsoft.AspNet.Hosting/Internal/RequestServicesContainerMiddleware.cs

```diff
@@ -43,10 +43,14 @@ public RequestServicesContainerMiddleware(RequestDelegate next, IServiceProvider
             // All done if request services is set
             if (existingFeature?.RequestServices != null)
```

Comment:
Depends if is skippable for benchmark I suppose? Stepping through profile traces line by line atm